test/parallel/test-child-process-spawn-name.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+// This test spawns itself with an argument to indicate when it is a child to
+// easily and portably print the process title
+if (process.argv.length == 3) {
+  console.log(process.title);
+  process.exit();
+}
+
+const noCustomName = cp.spawn('node',
```

Comment:
I don't think I can do that in this test, since the check needs to compare against the name which will be supplied as `argv[0]`, which is this parameter. If the test uses `execPath`, it will vary based the machine it's running on.